src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -0,0 +1,194 @@
+        label="Barcode:"
+        label-for="barcode_input"
+        description="Please scan the single barcode you wish to create labels for."
+      >
+        <b-form-input
+          id="barcode_input"
+          v-model="form.barcode"
+          placeholder="Please scan the barcode"
+          required
+        ></b-form-input>
+      </b-form-group>
+
+      <b-form-group
+        id="suffix_selection_group"
+        label="Suffix:"
+        label-for="suffix_selection"
+        description="Please select the suffix you want to use from the list."
+      >
+        <b-form-select
+          id="suffix_selection"
+          v-model="form.selectedSuffixId"
+          :options="suffixOptions"
+          placeholder="Please select a suffix"
+          required
+        ></b-form-select>
+      </b-form-group>
+
+      <b-form-group
+        id="number_of_labels_group"
+        label="Number of labels:"
+        label-for="number_of_labels"
+        description="Please enter the number of labels to make (with incrementing suffixes). Up to 9 to ensure printing"
+      >
+        <b-form-input
+          id="number_of_labels"
+          v-model="form.selectedNumberOfLabels"
+          type="number"
+          :min="1"
+          :max="9"
+          placeholder="Please enter a number"
+          required
+        ></b-form-input>
+      </b-form-group>
+
+      <b-form-group
+        id="printer_choice_group"
+        label="Choice of Printer:"
+        label-for="printer_choice"
+        description="Please select which printer you wish to use to print the labels."
+      >
+        <b-form-select
+          id="printer_choice"
+          v-model="form.selectedPrinterId"
+          :options="printerOptions"
+          placeholder="Please select a printer"
+          required
+        ></b-form-select>
+      </b-form-group>
+
+      <b-form-group
+        id="copies_group"
+        label="Number of copies per label:"
+        label-for="copies"
+        description="Please select how many copies of each label you would like printing."
+      >
+        <b-form-input
+          id="copies"
+          v-model="form.copies"
+          type="number"
+          :min="1"
+          :max="10"
+          placeholder="Please select a number"
+          required
+        ></b-form-input>
+      </b-form-group>
+
+      <b-button id="resetButton" type="reset" variant="danger" class="float-left">Reset</b-button>
+
+      <labelPrintingModal
+        id="labelPrintingModal"
+        ref="labelPrintingModal"
+        class="float-right"
+        :disabled="!formValid"
+        v-bind="propsToPass()"
+      ></labelPrintingModal>
+    </b-form>
+  </div>
+</template>
+
+<script>
+import LabelPrintingModal from '@/components/labelPrinting/LabelPrintingModal'
+const MESSAGE_SELECT = 'Please select a option'
+
+export default {
+  name: 'LabelPrintingForm',
+  components: {
+    LabelPrintingModal,
+  },
+  data() {
+    return {
+      form: {
```

Comment:
Mabe if the move the creation of the empty form to a method which creates a new object e.g. 
```
newForm = { barcode: null,
selectedSuffixId: null,
selectedNumberOfLabels: null,
selectedPrinterId: null,
copies: null }
```

then when you do reset you could just recreate the object.